app/helpers/sinicum/helper_utils.rb

```diff
@@ -149,16 +149,32 @@ def workspace(options)
       options[:workspace] || DEFAULT_DOCUMENT_WORKSPACE
     end
 
-    #srcset optimization, see http://w3c.github.io/html/semantics-embedded-content.html#element-attrdef-img-srcset
-    def add_srcset(attributes_hash)
+    def original_picture_width(src_attribute)
```

Comment:
This is too hacky for me... maybe you can get the original width from the attributes.